### DISCUSSION TITLE
MOV playback issue

### DISCUSSION THREAD

**Discussion by timwood3166** [smplayer-dev/smplayer#4] [Q&A]
I recorded an .mov with my webcam. SMPlayer will not play the video. Media player, and VLC will. And now, the details...
.mov
h.264
717 MB
3840x2160 30fps

Windows 10, AMD Ryzen 9 3900x, RTX 2080Ti

Any ideas?

**Comment by smplayer-dev** [comment]
Send the MPlayer log after trying to play the video.

**Comment by dandv** [comment]
@timwood3166: "MOV playback issue" is pretty vague, please be more specific in the discussion title, e.g. "Playback won't start for MOV recorded with webcam". Thanks :)
